### PR TITLE
x11-misc/xcwd: new package

### PR DESCRIPTION
**Pull request by mrabe89**
Hi.

xcwd is a simple tool that prints the current working directory of the currently focused window.
I used it on my arch system for years an dearly missed it on my gentoo setup.

This is my first gentoo PR, so please let me know if I need to change sth.
Thank you.

### CODE REVIEW

**Comment by Alessandro-Barbieri** [comment]
You can also add this to https://wiki.gentoo.org/wiki/Project:GURU in the meanwhile

File: x11-misc/xcwd/xcwd-20190309.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+RDEPEND="
+	x11-libs/libX11
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	${DEPEND}
+	virtual/pkgconfig
+"
+src_install() {
+	mkdir -p "${D}"/usr/bin
```
[the remark this answers is not in the document]
Comment (mrabe89) [reply x11-misc/xcwd/xcwd-20190309.ebuild:25]: Is there a better way to do this? - The following emake install fails if that directory doesn't exist.

File: x11-misc/xcwd/xcwd-20190309.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	x11-libs/libX11
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
```
[the remark this answers is not in the document]
Comment (mrabe89) [reply x11-misc/xcwd/xcwd-20190309.ebuild:20]: Ok, that's true for pkgconfig, or am I missing something?

**Comment by ajakk** [comment]
New leaf packages aren't added via proxy-maint anymore, but please add to the GURU overlay if still interested!

**Pull request closed without merging**